2022-01-12 17:22:41,666:INFO:Welcome to csv func
2022-01-12 17:22:41,930:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:22:42,171:INFO:Creating table....
2022-01-12 17:22:42,261:INFO:Table is created....
2022-01-12 17:22:42,269:ERROR:ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '1
isHighlight                                                     False
accessio' at line 1
2022-01-12 17:26:51,995:INFO:Welcome to csv func
2022-01-12 17:26:52,436:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:26:52,621:INFO:Creating table....
2022-01-12 17:26:52,695:INFO:Table is created....
2022-01-12 17:26:52,719:ERROR:ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '1
isHighlight                                                     False
accessio' at line 1
2022-01-12 17:30:14,519:INFO:Welcome to csv func
2022-01-12 17:30:14,739:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:30:14,899:INFO:Creating table....
2022-01-12 17:30:14,989:INFO:Table is created....
2022-01-12 17:30:15,003:ERROR:ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '1
isHighlight                                                     False
accessio' at line 1
2022-01-12 17:33:01,520:INFO:Welcome to csv func
2022-01-12 17:34:04,070:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:34:13,515:INFO:Creating table....
2022-01-12 17:34:19,530:INFO:Table is created....
2022-01-12 17:38:30,737:INFO:Welcome to csv func
2022-01-12 17:38:30,984:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:38:31,149:INFO:Creating table....
2022-01-12 17:38:31,212:INFO:Table is created....
2022-01-12 17:38:31,451:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-12 17:42:40,069:INFO:Welcome to csv func
2022-01-12 17:42:58,415:INFO:You're connected to database: ('csvfile_upload',)
2022-01-12 17:42:59,848:INFO:Creating table....
2022-01-12 17:43:01,456:INFO:Table is created....
2022-01-12 17:43:21,407:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-13 09:15:34,373:ERROR:PermissionError: [Errno 13] Permission denied: 'file.csv'
2022-01-13 09:18:34,108:ERROR:PermissionError: [Errno 13] Permission denied: 'file.csv'
2022-01-13 09:22:27,646:ERROR:UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 22: character maps to <undefined>
2022-01-13 09:24:03,870:INFO:Welcome to csv func
2022-01-13 09:24:04,133:INFO:You're connected to database: ('csvfile_upload',)
2022-01-13 09:24:04,357:INFO:Creating table....
2022-01-13 09:24:04,449:INFO:Table is created....
2022-01-13 09:24:05,085:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-13 09:37:20,631:INFO:Welcome to csv func
2022-01-13 09:37:20,925:INFO:You're connected to database: ('csvfile_upload',)
2022-01-13 09:37:21,152:INFO:Creating table....
2022-01-13 09:37:21,325:INFO:Table is created....
2022-01-13 09:37:21,676:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-13 09:59:26,572:INFO:Welcome to csv func
2022-01-13 09:59:26,830:INFO:You're connected to database: ('csvfile_upload',)
2022-01-13 09:59:26,995:INFO:Creating table....
2022-01-13 09:59:27,068:INFO:Table is created....
2022-01-13 09:59:27,384:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-15 11:57:34,564:INFO:Welcome to csv func
2022-01-15 11:57:34,771:INFO:You're connected to database: ('csvfile_upload',)
2022-01-15 11:57:34,859:INFO:Creating table....
2022-01-15 11:57:34,912:INFO:Table is created....
2022-01-15 11:57:35,149:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-24 15:56:08,286:INFO:Welcome to csv func
2022-01-24 15:56:08,745:INFO:You're connected to database: ('csvfile_upload',)
2022-01-24 15:56:09,175:INFO:Creating table....
2022-01-24 15:56:09,371:INFO:Table is created....
2022-01-24 15:56:09,813:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 02:07:02,804:INFO:Welcome to csv func
2022-01-26 02:07:02,970:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 02:07:03,165:INFO:Creating table....
2022-01-26 02:07:03,231:INFO:Table is created....
2022-01-26 02:07:03,462:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 02:10:01,021:INFO:Welcome to csv func
2022-01-26 02:10:01,410:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 02:10:01,549:INFO:Creating table....
2022-01-26 02:10:01,609:INFO:Table is created....
2022-01-26 02:10:01,900:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 15:46:03,685:ERROR:ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2022-01-26 16:23:05,222:INFO:Welcome to csv func
2022-01-26 16:23:05,507:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 16:23:05,612:INFO:Creating table....
2022-01-26 16:23:05,666:INFO:Table is created....
2022-01-26 16:23:05,887:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 16:23:56,647:INFO:Welcome to csv func
2022-01-26 16:23:56,804:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 16:23:56,868:INFO:Creating table....
2022-01-26 16:23:56,911:INFO:Table is created....
2022-01-26 16:23:57,147:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 16:24:20,172:INFO:Welcome to csv func
2022-01-26 16:24:20,201:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 16:24:20,229:INFO:Creating table....
2022-01-26 16:24:20,260:INFO:Table is created....
2022-01-26 16:24:20,479:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 16:25:02,994:INFO:Welcome to csv func
2022-01-26 16:25:03,066:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 16:25:03,100:INFO:Creating table....
2022-01-26 16:25:03,135:INFO:Table is created....
2022-01-26 16:25:03,333:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 17:51:09,463:INFO:Welcome to csv func
2022-01-26 17:51:09,654:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 17:51:09,787:INFO:Creating table....
2022-01-26 17:51:09,843:INFO:Table is created....
2022-01-26 17:51:10,147:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 17:51:19,472:ERROR:OperationalError: (20002, b'DB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\nDB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\n')
2022-01-26 17:54:23,517:INFO:Welcome to csv func
2022-01-26 17:54:23,718:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 17:54:23,852:INFO:Creating table....
2022-01-26 17:54:23,897:INFO:Table is created....
2022-01-26 17:54:24,187:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 17:54:30,674:ERROR:OperationalError: (20002, b'DB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\nDB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\n')
2022-01-26 17:55:46,677:INFO:Welcome to csv func
2022-01-26 17:55:46,899:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 17:55:46,998:INFO:Creating table....
2022-01-26 17:55:47,055:INFO:Table is created....
2022-01-26 17:55:47,329:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 18:10:09,599:INFO:Welcome to csv func
2022-01-26 18:10:09,786:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 18:10:09,911:INFO:Creating table....
2022-01-26 18:10:09,953:INFO:Table is created....
2022-01-26 18:10:10,210:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 18:12:38,766:INFO:Welcome to csv func
2022-01-26 18:12:38,929:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 18:12:39,034:INFO:Creating table....
2022-01-26 18:12:39,084:INFO:Table is created....
2022-01-26 18:12:39,375:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 18:25:58,428:INFO:Welcome to csv func
2022-01-26 18:25:58,600:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 18:25:58,717:INFO:Creating table....
2022-01-26 18:25:58,764:INFO:Table is created....
2022-01-26 18:25:59,067:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 18:26:15,590:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-26 18:35:02,249:INFO:Welcome to csv func
2022-01-26 18:35:02,411:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 18:35:02,503:INFO:Creating table....
2022-01-26 18:35:02,544:INFO:Table is created....
2022-01-26 18:35:02,811:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 18:35:21,583:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-26 19:11:24,925:INFO:Welcome to csv func
2022-01-26 19:11:25,247:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 19:11:25,520:INFO:Creating table....
2022-01-26 19:11:25,668:INFO:Table is created....
2022-01-26 19:11:26,113:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 19:11:37,279:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-26 19:59:00,506:INFO:Welcome to csv func
2022-01-26 19:59:00,852:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 19:59:01,063:INFO:Creating table....
2022-01-26 19:59:01,141:INFO:Table is created....
2022-01-26 19:59:01,566:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 19:59:09,103:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-26 20:54:27,142:INFO:Welcome to csv func
2022-01-26 20:54:27,390:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 20:54:27,590:INFO:Creating table....
2022-01-26 20:54:27,670:INFO:Table is created....
2022-01-26 20:54:27,993:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 20:54:34,628:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-26 23:12:50,883:INFO:Welcome to csv func
2022-01-26 23:12:51,230:INFO:You're connected to database: ('csvfile_upload',)
2022-01-26 23:12:51,571:INFO:Creating table....
2022-01-26 23:12:51,701:INFO:Table is created....
2022-01-26 23:12:52,111:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-26 23:12:52,174:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-01-27 09:31:52,352:INFO:Welcome to csv func
2022-01-27 09:31:52,622:INFO:You're connected to database: ('csvfile_upload',)
2022-01-27 09:31:52,804:INFO:Creating table....
2022-01-27 09:31:52,871:INFO:Table is created....
2022-01-27 09:31:53,170:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-01-27 09:31:53,299:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-01 00:38:17,671:INFO:Welcome to csv func
2022-02-01 00:38:17,984:INFO:You're connected to database: ('csvfile_upload',)
2022-02-01 00:38:18,268:INFO:Creating table....
2022-02-01 00:38:18,365:INFO:Table is created....
2022-02-01 00:38:18,636:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-02-01 00:38:18,721:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-01 00:41:08,878:INFO:Welcome to csv func
2022-02-01 00:41:09,056:INFO:You're connected to database: ('csvfile_upload',)
2022-02-01 00:41:09,186:INFO:Creating table....
2022-02-01 00:41:09,260:INFO:Table is created....
2022-02-01 00:41:09,526:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-02-01 00:41:09,584:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-01 01:07:53,015:INFO:Welcome to csv func
2022-02-01 01:07:53,186:INFO:You're connected to database: ('csvfile_upload',)
2022-02-01 01:07:53,336:INFO:Creating table....
2022-02-01 01:07:53,401:INFO:Table is created....
2022-02-01 01:07:53,799:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-02-01 01:07:53,838:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:21:13,612:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:21:31,700:INFO:Welcome to csv func
2022-02-10 10:21:31,891:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:21:32,105:INFO:Creating table....
2022-02-10 10:21:32,174:INFO:Table is created....
2022-02-10 10:21:32,410:ERROR:DataError: 1406 (22001): Data too long for column 'primaryImage' at row 1
2022-02-10 10:30:56,033:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:31:10,184:INFO:Welcome to csv func
2022-02-10 10:31:10,306:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:31:10,422:INFO:Creating table....
2022-02-10 10:31:10,471:INFO:Table is created....
2022-02-10 10:31:10,505:ERROR:IntegrityError: 1062 (23000): Duplicate entry '0' for key 'csvfile_data.PRIMARY'
2022-02-10 10:40:52,856:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:44:43,133:INFO:Welcome to csv func
2022-02-10 10:44:43,351:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:44:43,476:INFO:Creating table....
2022-02-10 10:44:43,570:INFO:Table is created....
2022-02-10 10:44:43,617:ERROR:IntegrityError: 1062 (23000): Duplicate entry '0' for key 'csvfile_data.PRIMARY'
2022-02-10 10:47:31,160:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:47:50,691:INFO:Welcome to csv func
2022-02-10 10:47:50,811:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:47:50,935:INFO:Creating table....
2022-02-10 10:47:50,997:INFO:Table is created....
2022-02-10 10:47:51,014:ERROR:DataError: 1136 (21S01): Column count doesn't match value count at row 1
2022-02-10 10:48:11,880:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:48:33,785:INFO:Welcome to csv func
2022-02-10 10:48:33,869:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:48:33,955:INFO:Creating table....
2022-02-10 10:48:34,002:INFO:Table is created....
2022-02-10 10:48:34,013:ERROR:DataError: 1136 (21S01): Column count doesn't match value count at row 1
2022-02-10 10:50:31,125:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:51:02,513:INFO:Welcome to csv func
2022-02-10 10:51:02,648:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:51:02,743:INFO:Creating table....
2022-02-10 10:51:02,796:INFO:Table is created....
2022-02-10 10:51:02,807:ERROR:DataError: 1136 (21S01): Column count doesn't match value count at row 1
2022-02-10 10:52:04,734:INFO:Welcome to csv func
2022-02-10 10:52:04,757:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:52:04,797:INFO:Creating table....
2022-02-10 10:52:04,842:INFO:Table is created....
2022-02-10 10:52:04,849:ERROR:DataError: 1136 (21S01): Column count doesn't match value count at row 1
2022-02-10 10:55:38,203:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 10:55:50,179:ERROR:ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
2022-02-10 10:56:32,352:INFO:Welcome to csv func
2022-02-10 10:58:22,545:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 10:58:29,001:INFO:Creating table....
2022-02-10 10:58:39,930:INFO:Table is created....
2022-02-10 11:02:00,824:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:02:21,962:INFO:Welcome to csv func
2022-02-10 11:02:39,273:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:02:40,820:INFO:Creating table....
2022-02-10 11:02:42,757:INFO:Table is created....
2022-02-10 11:12:10,281:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:12:40,384:INFO:Welcome to csv func
2022-02-10 11:13:01,685:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:13:03,356:INFO:Creating table....
2022-02-10 11:13:05,028:INFO:Table is created....
2022-02-10 11:14:12,335:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:14:39,341:INFO:Welcome to csv func
2022-02-10 11:14:48,964:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:14:50,307:INFO:Creating table....
2022-02-10 11:14:51,432:INFO:Table is created....
2022-02-10 11:22:43,214:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:22:53,870:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:23:04,322:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:23:34,399:INFO:Welcome to csv func
2022-02-10 11:23:52,924:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:23:57,100:INFO:Creating table....
2022-02-10 11:24:02,496:INFO:Table is created....
2022-02-10 11:26:13,141:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:26:31,989:INFO:Welcome to csv func
2022-02-10 11:26:42,420:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:26:43,279:INFO:Creating table....
2022-02-10 11:26:44,826:INFO:Table is created....
2022-02-10 11:27:35,397:ERROR:IntegrityError: 1062 (23000): Duplicate entry '0' for key 'csvfile_data.PRIMARY'
2022-02-10 11:31:05,558:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:31:19,837:INFO:Welcome to csv func
2022-02-10 11:31:33,544:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:31:34,489:INFO:Creating table....
2022-02-10 11:31:35,691:INFO:Table is created....
2022-02-10 11:34:21,421:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:34:36,407:INFO:Welcome to csv func
2022-02-10 11:34:36,521:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:34:36,633:INFO:Creating table....
2022-02-10 11:34:36,688:INFO:Table is created....
2022-02-10 11:36:28,331:ERROR:KeyError: 'objectID'
2022-02-10 11:36:44,587:ERROR:KeyError: 'objectID'
2022-02-10 11:36:52,347:ERROR:KeyError: 'objectID'
2022-02-10 11:38:53,297:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:39:12,447:INFO:Welcome to csv func
2022-02-10 11:39:15,039:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:39:15,164:INFO:Creating table....
2022-02-10 11:39:15,227:INFO:Table is created....
2022-02-10 11:41:33,541:ERROR:KeyError: 'objectID'
2022-02-10 11:43:18,128:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:43:46,701:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:43:56,375:INFO:Welcome to csv func
2022-02-10 11:43:56,505:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:43:56,634:INFO:Creating table....
2022-02-10 11:43:56,694:INFO:Table is created....
2022-02-10 11:52:59,867:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 11:53:13,949:INFO:Welcome to csv func
2022-02-10 11:53:14,067:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 11:53:14,184:INFO:Creating table....
2022-02-10 11:53:14,236:INFO:Table is created....
2022-02-10 11:56:10,678:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 14:05:34,156:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 14:05:50,007:INFO:Welcome to csv func
2022-02-10 14:05:50,191:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 14:05:50,351:INFO:Creating table....
2022-02-10 14:05:50,415:INFO:Table is created....
2022-02-10 14:06:19,418:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 14:06:30,306:INFO:Welcome to csv func
2022-02-10 14:06:30,386:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 14:06:30,466:INFO:Creating table....
2022-02-10 14:06:30,522:INFO:Table is created....
2022-02-10 14:33:03,103:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 14:33:33,845:INFO:Welcome to csv func
2022-02-10 14:33:34,037:INFO:You're connected to database: ('csvfile_upload',)
2022-02-10 14:33:34,213:INFO:Creating table....
2022-02-10 14:33:34,269:INFO:Table is created....
2022-02-10 15:15:13,200:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 15:15:48,098:ERROR:TypeError: delete() takes 1 positional argument but 2 were given
2022-02-10 15:16:07,252:ERROR:TypeError: delete() takes 1 positional argument but 2 were given
2022-02-10 15:21:39,188:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 15:54:34,639:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 15:56:21,238:ERROR:TypeError: 'Response' object is not subscriptable
2022-02-10 15:57:39,876:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 16:07:50,307:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 16:11:56,751:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 16:20:40,109:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 16:24:45,237:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 16:26:17,607:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:28:35,917:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:40:49,090:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:46:14,976:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:47:46,225:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:47:58,650:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:48:02,438:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:48:08,118:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:48:16,013:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:50:09,182:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:51:05,526:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:51:12,847:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 18:52:09,957:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 18:53:52,122:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 19:05:20,812:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:11:56,518:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:23:13,631:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:24:49,402:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 19:27:57,940:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:32:10,304:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:35:20,727:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 19:35:43,295:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 19:35:58,275:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 19:36:22,541:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 19:38:03,655:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 21:52:26,194:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 21:55:18,374:ERROR:AttributeError: 'str' object has no attribute 'values'
2022-02-10 21:57:26,142:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:00:28,899:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:27:50,888:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:48:21,749:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:51:29,281:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:56:26,150:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 22:58:11,735:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 23:01:54,355:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 23:05:13,033:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 23:11:30,705:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-10 23:23:35,179:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-11 00:15:43,571:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-11 00:17:40,103:ERROR:KeyError: 'objectId'
2022-02-11 00:22:30,082:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-11 00:23:52,989:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-11 00:31:04,887:INFO:Found credentials in shared credentials file: ~/.aws/credentials
2022-02-11 00:32:52,795:INFO:Found credentials in shared credentials file: ~/.aws/credentials
